docs/redirects.py

```diff
@@ -290,12 +290,25 @@ def all_urls_from_files():
         exit(1)
 
     if sys.argv[1] == "check":
+        #check for broken links
         errors = check_links(links)
         if errors:
             print("check failed, errors detected!", file=sys.stderr)
             for e in errors:
                 print(e, file=sys.stderr)
             exit(1)
```

Comment:
is there a way to return all errors at once, instead of exiting before checking for dropped urls?

Reply:
Good point, I switched it to have a return_code variable.